src/System.Net.Http/src/System/Net/Http/Unix/CurlHandler.cs

```diff
@@ -121,7 +181,7 @@ private void StartRequest(RequestCompletionSource state)
                 return;
             }
 
-            var cancellationTokenRegistration = state.CancellationToken.Register(x => ((RequestCompletionSource) x).TrySetCanceled(), state);
+            var cancellationTokenRegistration = state.CancellationToken.Register(x => ((RequestCompletionSource)x).TrySetCanceled(), state);
```

Comment:
Nit: I believe this spacing-only change is going to cause merge conflicts with some of the other HttpClient changes being made in parallel.